routes/users.js

```diff
@@ -0,0 +1,8 @@
+var express = require('express');
+var router = express.Router();
+
+router.get('/', function(req, res, next) {
+  res.send('respond with a resource');
```

Comment:
is 'respond with a resource' a dev message? If so maybe put 'msg: it responds with a resource' so that is more apparent.